Week1/homework/todos.sql

```diff
@@ -0,0 +1,100 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
no problem using an sql generator just make sure you know whats going on !

Reply:
I used MySQL workbench to generate script but really it is very easy to me :) 👍 💯